test/MHD/sphBragTDiffusion/python/testidefix.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Apr 27 14:00:41 2021
+
+@author: jmkempf
+"""
+
```

Comment:
This is all absolutely superfluous (and the date isn't even correct)
```suggestion
```

Reply:
Deleted in the next commit